code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1057,6 +1058,25 @@
 	REMOVE_TRAIT(M, TRAIT_STABLEHEART, type)
 	REMOVE_TRAIT(M, TRAIT_STABLELIVER, type)
 	..()
+/datum/reagent/medicine/penthrite
+	// Same as corazone. but slightly better
+	name = "Penthrite"
```

Comment:
need to establish a suffix for organ-related meds. I don't care what it is, it just needs to be doc'd somewhere (preferably the C2 file with all the others).

Reply:
> need to establish a suffix for organ-related meds. I don't care what it is, it just needs to be doc'd somewhere (preferably the C2 file with all the others).

aight, tell me when you establish em. for now it will stay as it is.

Reply:
`rite` is fine, just update the C2 file to include the prefix